packages/cascara/src/ui/Table/TableBase.js

```diff
@@ -55,9 +57,64 @@ const TableBase = ({
   dataConfig,
   dataDisplay,
   onAction,
+  selections,
   uniqueIdAttribute,
   ...rest
 }) => {
+  // Row selection
+  const isRowSelectable = Boolean(selections);
+  const maxSelection = isRowSelectable ? selections?.max || 0 : 0;
+  const [selection, updateSelection] = useReducer(selectionReducer, []);
+
+  // get a list of record IDs
+  const recordIDs = useMemo(
+    () =>
+      data && uniqueIdAttribute
+        ? data.map((record) => record[uniqueIdAttribute])
+        : [],
+    [data, uniqueIdAttribute]
+  );
+
+  /**
+   * Select row(s)
+   *
+   * @param {String|Array[String]} rowID
+   */
+  const select = useCallback(
```

Comment:
```suggestion
  const rowSelect = useCallback(
```
I think it would be helpful to name these functions with more detailed descriptions than just `select`. What are we selecting? Additionally, since we should try to sort our methods alphabetically within the closure, it makes sense to try to name our functions in a way that communicates relationships between them.

The related function for this I think is the `unselect` function which could be renamed `rowUnselect` to be more clear what it does, and then they have similar name patterns for being sorted.